addons/api/mirage/config.js

```diff
@@ -368,36 +368,36 @@ export default function () {
         version: attrs.version,
       };
       // If adding host sets, push them into the array
-      if (method === 'add-host-sets') {
+      if (method === 'add-host-sources') {
         updatedAttrs.hostSetIds = target.hostSetIds;
-        attrs.hostSetIds.forEach((id) => {
+        attrs.hostSourceIds.forEach((id) => {
           if (!updatedAttrs.hostSetIds.includes(id)) {
             updatedAttrs.hostSetIds.push(id);
           }
         });
       }
       // If deleting host sets, filter them out of the array
-      if (method === 'remove-host-sets') {
+      if (method === 'remove-host-sources') {
         updatedAttrs.hostSetIds = target.hostSetIds;
         updatedAttrs.hostSetIds = updatedAttrs.hostSetIds.filter((id) => {
```

Comment:
updatedAttrs.hostSetIds = updatedAttrs.hostSetIds.filter --> updatedAttrs.hostSourceIds = updatedAttrs.hostSourceIds.filter

Reply:
The attribute in mocks remains `hostSets` for now.